_posts/2025-02-25-cpansec-is-cna-for-perl-and-cpan.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+toc: false
+title: "CPANSec is CNA for Perl and the CPAN ecosystem"
+date: 2025-02-25 22:00:00 +0100
+tags: cpan perl cve cna
+author:  Stig Palmquist, Timothy Legge and Breno G. de Oliveira
+---
+
+The CPAN Security Group was [authorized by the CVE Program](https://www.cve.org/Media/News/item/news/2025/02/25/CPAN-Security-Group-Added-as-CNA) as a CVE Numbering Authority (CNA) on Feb 25, 2025. A CNA assigns and manages CVE identifiers for projects in their scope.
+
+Our scope is vulnerabilities in Perl and CPAN Modules (including End-of-Life Perl versions) found at [perl.org](https://perl.org/), [cpan.org](https://cpan.org/) or [metacpan.org](https://metacpan.org/), excluding distributions of Perl or CPAN Modules maintained by third-party redistributors.
```

Comment:
s/Our scope/The CPANSec CNA scope/ maybe?

Also maybe s/found/made available through/

Finally, I am not sure what "distributions of Perl or CPAN Modules maintained by third-parties redistributors" mean. Is it activeperl and the like? packages from linux distros? And I wonder if we should reinforce somehow that modules and distributions that were never hosted by those domains are also excluded.

Reply:
Yes, that is the intention.  ActiveState, https://velociperl.com/ for instance would be out of scope

Also, Perl Applications like foswiki.org is out of scope.  I never really thought of the packages in Linux distros but they are definitely out of scope